sql/updates/latest-dev.sql

```diff
@@ -89,3 +89,185 @@ AS '@MODULE_PATHNAME@', 'ts_update_placeholder' LANGUAGE C IMMUTABLE STRICT PARA
 
 CREATE OR REPLACE FUNCTION _timescaledb_functions.get_internal_time_max(REGTYPE) RETURNS BIGINT
 AS '@MODULE_PATHNAME@', 'ts_update_placeholder' LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
+
+CREATE FUNCTION _timescaledb_functions.get_materialization_info(
+    continuous_aggregate REGCLASS
+) RETURNS RECORD AS
+$body$
+DECLARE
+  info RECORD;
+BEGIN
+    SELECT mat_hypertable_id AS materialization_id,
+           bucket_width::interval AS bucket_width
+      INTO info
+      FROM _timescaledb_catalog.continuous_agg
+      JOIN _timescaledb_catalog.continuous_aggs_bucket_function
+     USING (mat_hypertable_id)
+     WHERE format('%I.%I', user_view_schema, user_view_name)::regclass = continuous_aggregate;
+
+    IF NOT FOUND THEN
+        RAISE '"%" is not a continuous aggregate', continuous_aggregate
+        USING ERRCODE = 'wrong_object_type';
+    END IF;
+
+    RETURN info;
+END
+$body$
+LANGUAGE plpgsql
+SET search_path TO pg_catalog, pg_temp;
```

Comment:
I don't think it is necessary to add those to latest-dev.sql since the cagg_api.sql file will be part of the update script anyway.

Reply:
Hmm... previously, PGSpot complained if you did not, but it seems like it doesn't any more.